web/packages/teleport/src/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration.story.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { http, HttpResponse } from 'msw';
+import { MemoryRouter } from 'react-router';
+
+import { InfoGuidePanelProvider } from 'shared/components/SlidingSidePanel/InfoGuide';
+
+import cfg from 'teleport/config';
+import { ContextProvider } from 'teleport/index';
+import { IamIntegration } from 'teleport/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration';
+import { makeAwsOidcStatusContextState } from 'teleport/Integrations/status/AwsOidc/testHelpers/makeAwsOidcStatusContextState';
+import { MockAwsOidcStatusProvider } from 'teleport/Integrations/status/AwsOidc/testHelpers/mockAwsOidcStatusProvider';
+import { createTeleportContext } from 'teleport/mocks/contexts';
+
+export default {
+  title: 'Teleport/Integrations/Enroll/AwsConsole',
+};
+
+const ctx = createTeleportContext();
+
+// For success view use:
+// * Integration Name: test
+// * Trust Anchor, Profile and Role ARNs - any valid input, ex:
+//   arn:aws:rolesanywhere:eu-west-2:123456789012:trust-anchor/foo
+//   arn:aws:rolesanywhere:eu-west-2:123456789012:profile/bar
+//   arn:aws:iam::123456789012:role/baz
```

Comment:
nice 👍 it would be better to put this on a `<Info>` banner though (folks have been appreciative of it too)
```suggestion
// For success view use:
// * Step 1: Integration Name: test
// * Step 3: Trust Anchor, Profile and Role ARNs - any valid input, ex:
//   arn:aws:rolesanywhere:eu-west-2:123456789012:trust-anchor/foo
//   arn:aws:rolesanywhere:eu-west-2:123456789012:profile/bar
//   arn:aws:iam::123456789012:role/baz
```